docs/core/stories.rst

```diff
@@ -49,13 +52,73 @@ This is what we call a **story**.
   in the format ``intent{"entity1": "value", "entity2": "value"}``.
 - Actions executed by the bot are shown as lines starting with ``-``
   and contain the name of the action.
-- Events returned by an action are on lines immediately after that
-  action. For example, if an action returns a ``SetSlot`` event,
-  this is shown as the line ``- slot{"slot_name": "value"}``
+- Events returned by an action are on lines immediately after that action.
+  For example, if an action returns a ``SlotSet`` event, this is shown as
+  ``slot{"slot_name": "value"}``.
+
+Let's now take a slightly more detailed look at each of these components,
```

Comment:
This is kind of blog-posty. Maybe something like "more detailed information about each of these components and how to use them follows" but even as i'm writing this, feels like we should just get rid of this paragraph